lib/countess-sys.js

```diff
@@ -11,7 +11,7 @@ export default class CounteSS {
   // Returns an object that can be retrieved when package is activated
   serialize() {}
 
-  analyze() {
+  analyze(callback) {
```

Comment:
We can leave the callback capability in for now, but maybe in the future we can remove it if it never gets used.